app/models/vote.rb

```diff
@@ -90,6 +90,7 @@ def associate_previous_vote
   end
 
   def age_previous_votes
+    Vote.connection.execute("SET CONSTRAINTS vote_age_per_user_per_motion DEFERRED")
```

Comment:
We already have a migration that ensures the constraint is deferrable until commit. By default, however, it is still immediate. This line will defer the constraint only within the current transaction hence it's in application code.